testing/kokoro/configure_gcloud.bash

```diff
@@ -30,6 +30,7 @@ fi
 gcloud -q components update
 gcloud -q components install app-engine-go
 gcloud -q components install beta # Install beta component needed for Cloud Run E2E tests
+gcloud -q components install alpha # Install beta component needed for Cloud Run E2E tests
```

Comment:
It would be good to know why we need the alpha component, perhaps another comment including specific flags?

```suggestion
gcloud -q components install alpha # Install alpha component needed for Cloud Run E2E tests
```